lib/cli/src/frameworks/react/ts/Header.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { Header } from './Header';
+
+export default {
+  title: 'Example/Header',
+  component: Header,
+};
+
+const Template = (args) => <Header {...args} />;
+
+export const LoggedIn = Template.bind({});
+LoggedIn.args = {
+  user: {},
+};
+
+export const LoggedOut = Template.bind({});
+LoggedOut.args = {};
```

Comment:
Is the empty args here just to show people they can use it or is it necessary? There's already an example in `LoggedIn`, so no need to add it, right? It might lead to people thinking that they always need to add an empty args to every story